janusgraph-cassandra/src/main/java/org/janusgraph/diskstorage/cassandra/AbstractCassandraStoreManager.java

```diff
@@ -332,4 +332,9 @@ public String getName() {
         return getClass().getSimpleName() + keySpaceName;
     }
 
+    // Opened up for testing purposes
+    protected String setKeyspaceName(Configuration config) {
```

Comment:
This doesn't set anything.  Should be renamed according to what it does. `determineKeyspaceName`?